frontend/packages/client/src/components/Proposal/RankedChoiceVote/TestCard.js

```diff
@@ -0,0 +1,69 @@
+import { useRef } from 'react';
+import { useDrag, useDrop } from 'react-dnd';
+
+const style = {
+  borderRadius: '40px',
+  backgroundColor: 'rgba(255,255,255,0) !important',
+  cursor: 'grab',
+  boxShadow: 'none !important',
+};
+export const Card = ({ id, children, index, moveCard, itemType }) => {
```

Comment:
Should the filename be `Card.js` instead of `TestCard.js`?

Reply:
Ah yes, this is not needed. I was using this from the example code to debug why my code was not working. Removed.